src/connectors/tapes-kglw.ts

```diff
@@ -0,0 +1,27 @@
+export {};
+
+Connector.playerSelector = '#site-header';
+
+Connector.getArtist = () => {
+	return 'King Gizzard & The Lizard Wizard';
+};
+
+Connector.trackSelector =
+	'#site-header div#now-playing > span#now-playing-track-title';
+
+Connector.albumSelector =
+	'#site-header div#now-playing > span#now-playing-show-date';
+
+Connector.currentTimeSelector =
+	'#site-header div#now-playing > span.now-playing-elapsed';
+
+Connector.durationSelector =
+	'#site-header div#now-playing > span.now-playing-duration hidden-sm';
+
+Connector.playButtonSelector = '#play-btn';
```

Comment:
I don't think this is needed as you're overriding the isPlaying function below.

Reply:
Good to know, updated.

Reply:
The duration one could have been left in I think but perhaps could be updated to 


```
Connector.durationSelector =
	'#now-playing > span.now-playing-duration';
```